src/material/form-field/testing/form-field-harness.ts

```diff
@@ -236,7 +236,9 @@ export class MatFormFieldHarness extends _MatFormFieldHarnessBase<FormFieldContr
   protected _suffixContainer = this.locatorForOptional('.mat-form-field-suffix');
   protected _label = this.locatorForOptional('.mat-form-field-label');
   protected _errors = this.locatorForAll('.mat-error');
-  protected _hints = this.locatorForAll('mat-hint, .mat-hint');
+  protected _hints = this.locatorForAll(
+    '.mat-form-field-hint-wrapper-active mat-hint, .mat-form-field-hint-wrapper-active .mat-hint',
```

Comment:
You should be able to reduce this to `.mat-form-field-hint-wrapper-active .mat-hint`. The tag name selector still adds the `mat-hint` class.

Reply:
Updated!